mindee/client.py

```diff
@@ -211,9 +211,9 @@ def enqueue_and_parse(
         page_options: Optional[PageOptions] = None,
         cropper: bool = False,
         endpoint: Optional[Endpoint] = None,
-        initial_delay_sec: float = 6,
-        delay_sec: float = 3,
-        max_retries: int = 10,
+        initial_delay_sec: float = 4,
+        delay_sec: float = 2,
+        max_retries: int = 30,
```

Comment:
30 seems a bit excessive, doesn't it? Might cause 429 on CI as well if we run multiple pushes

Reply:
No, 429 is not triggered from GET calls to check on progress. But we did have a user failing on large files at 10.

30x2= 60 seconds is not all that much really